helm-push/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apk add --no-cache bash git curl jq
 ENV XDG_DATA_HOME /root/.local/share
 ENV HELM_HOME ${XDG_DATA_HOME}/helm
 
-RUN helm2 init --client-only
+RUN helm2 init --client-only --stable-repo-url "https://charts.helm.sh/stable"
```

Comment:
do we even need `helm2` at this point?  might be best to just remove it

Reply:
Jordan and I were kind of wondering the same thing, but we weren't sure if there was a specific reason we were doing it this way in the first place. It would be as simple as removing that line, right?